assets/js/app.js

```diff
@@ -11,11 +11,12 @@
 //
 // If you no longer want to use a dependency, remember
 // to also remove its path from "config.paths.watched".
-import "phoenix_html"
+import 'phoenix_html'
```

Comment:
Missing semicolon.

[Read more about it here.](http://eslint.org/docs/rules/semi)